### PR TITLE
Add some internal tools to the list

### PR DESCRIPTION
**Pull request by ilyapuchka**

### CODE REVIEW

File: Cookbook/Technical-Documents/ToolsAndServices.md [hunk 1]
```diff
@@ -25,3 +16,31 @@
...
+* [Charles](https://www.charlesproxy.com) is a tool that allows to intercept network trafic from the app
+
+  * Install Charles Proxy
+  * Add the desired URL under Proxy > SSL Proxy Settings
+  * Install on Simulator via Help > SSL Proxying > Install on iOS Simulator
+  * The license for Charles can be found in the iOS team vault on 1Password
+
+* [Insomnia](https://insomnia.rest) is a REST client app that is helpful when working with APIs
+
+* [Fork](https://forkapp.io) and [GitUp](https://gitup.co) are the git client apps we love
```
Comment (daniloaliberti) [inline Cookbook/Technical-Documents/ToolsAndServices.md]: why not SourceTree ?

Comment (ilyapuchka) [reply Cookbook/Technical-Documents/ToolsAndServices.md]: Because it says "apps we love" 🤣 you should really try Fork instead, trust me you'll love it 👍

Comment (daniloaliberti) [reply Cookbook/Technical-Documents/ToolsAndServices.md]: ok then create the section `Personal Preference` here 😛

**Review by daniloaliberti — APPROVED** [review]

File: Cookbook/Technical-Documents/ToolsAndServices.md (hunk 1 above)
Comment (mluisbrown) [reply Cookbook/Technical-Documents/ToolsAndServices.md]: [GitUp](https://gitup.co) or GTFO 😂

Comment (diegopetrucci) [inline Cookbook/Technical-Documents/ToolsAndServices.md]:
```suggestion
* [Fork](https://forkapp.io) and [GitUp](https://gitup.co) are examples of git clients that some members of the team love```

Comment (RuiAAPeres) [reply Cookbook/Technical-Documents/ToolsAndServices.md]: Source Tree should be included in this list. I like it, and so do other members.

**Comment by mluisbrown** [comment]
[Knuff](https://github.com/KnuffApp/Knuff) and [Houston](https://github.com/nomad/houston) are useful for testing push notifications. We need a doc here on how to test them (currently it's only in the Wiki)

File: Cookbook/Technical-Documents/ToolsAndServices.md [hunk 2]
```diff
@@ -25,3 +16,31 @@
...
+* [Charles](https://www.charlesproxy.com) is a tool that allows to intercept network trafic from the app
+
+  * Install Charles Proxy
+  * Add the desired URL under Proxy > SSL Proxy Settings
+  * Install on Simulator via Help > SSL Proxying > Install on iOS Simulator
+  * The license for Charles can be found in the iOS team vault on 1Password
+
+* [Insomnia](https://insomnia.rest) is a REST client app that is helpful when working with APIs
+
+* [Fork](https://forkapp.io) and [GitUp](https://gitup.co) are examples of git clients that some members of the team love
```
Comment (daniloaliberti) [inline Cookbook/Technical-Documents/ToolsAndServices.md]:
```suggestion
* [Fork](https://forkapp.io), [GitUp](https://gitup.co) and [SourceTree](https://www.sourcetreeapp.com) are examples of git clients that some members of the team love
```

File: Cookbook/Technical-Documents/ToolsAndServices.md (hunk 1 above)
Comment (ilyapuchka) [reply Cookbook/Technical-Documents/ToolsAndServices.md]: Ok, ok, just because it's a good wether today!

**Comment by ilyapuchka** [comment]
@mluisbrown that's my next step, already have a draft PR for Troubleshooting QA doc for this kind of stuff 👍

**Review by MichalTKwiecien — APPROVED** [review]
😍

**Pull request merged**